ml_pipeline_engine/node/retrying.py

```diff
@@ -2,13 +2,13 @@
 from typing import Tuple
 from typing import Type
 
-from ml_pipeline_engine.types import NodeLike
+from ml_pipeline_engine.types import NodeBase
 from ml_pipeline_engine.types import RetryPolicyLike
 
 
 @dataclass(frozen=True)
 class NodeRetryPolicy(RetryPolicyLike):
```

Comment:
We need to eliminate this class because it's useless and we can work with nodes directly. Let's just make a new issue from the comment

Reply:
Did I understand you correctly that because we have a `RetryProtocol` that we use for each node, we don't need the `NodeRetryPolicy` or `RetryPolicyLike` ?

Reply:
yes

Reply:
done https://github.com/tochka-public/ml-pipeline-engine/issues/48